airflow/providers/google/cloud/operators/cloud_storage_transfer_service.py

```diff
@@ -547,22 +547,27 @@ def __init__(
         **kwargs,
     ) -> None:
         super().__init__(**kwargs)
-        self.filter = request_filter
+        self.request_filter = request_filter
         self.project_id = project_id
         self.gcp_conn_id = gcp_conn_id
         self.api_version = api_version
         self.google_impersonation_chain = google_impersonation_chain
 
+    @property
```

Comment:
Did you overlook the suggested `@deprecated` decorator?

Reply:
Yes, I realized that's how the parameter is documented in Google's docs.

Reply:
@okirialbert can you make the change?

Reply:
> @okirialbert can you make the change?

It should be ok without the deprecation, see discussion above.